packages/react/src/components/Accordion/Accordion.module.css

```diff
@@ -49,6 +48,10 @@
   margin: 0;
 }
 
+.border .accordionButton {
+  border-radius: var(--fdsc-accordion-border-radius);
+}
+
 .header > button {
```

Comment:
Haha, suggest also updating this to the new `.accordionButton` 😋

Reply:
Changed all of them 💯